core/src/input/input.tsx

```diff
@@ -28,22 +28,53 @@ const getClass = (props: InputProps): string => {
 };
 
 export interface InputProps {
+	/**
+	 * Type of input. For example: button, email, password, number, etc...
+	 */
 	type?: string;
 	// Uncontrolled
+
+	/**
+	 * Initial value of the input in uncontrolled mode
+	 */
 	defaultValue?: string | number;
+
+	/**
+	 * Usually useful in uncontrolled mode.
+	 * Read [Forwarding Refs](https://reactjs.org/docs/forwarding-refs.html) for details.
+	 */
 	forwardedRef?: React.ForwardedRef<HTMLInputElement>;
 	// Controlled
+
+	/**
+	 * Value of the select in controlled mode
+	 */
 	value?: string;
+
+	/**
+	 * Handler to set the value in controlled mode
+	 */
 	setValue?: (value: string) => void;
+
 	/**
 	 * Id of a datalist element to be used. Can pass an object with values for
```

Comment:
Should have an MDN link to the "datalist" keyword. Not many people know about it